src/conf.rs

```diff
@@ -0,0 +1,27 @@
+use ::springql_core::api::SpringConfig as RawSpringConfig;
```

Comment:
You need to add the following header line to all the new files.

```rust
// This file is part of https://github.com/SpringQL/SpringQL-client-c which is licensed under MIT OR Apache-2.0. See file LICENSE-MIT or LICENSE-APACHE for full license details.
```

See: https://github.com/SpringQL/SpringQL-client-c/runs/6767648263?check_suite_focus=true

Reply:
File name: How about renaming into `spring_config.rs`?